extensions/git/src/repository.ts

```diff
@@ -2055,6 +2060,9 @@ export class Repository implements Disposable {
 		// set count badge
 		this.setCountBadge();
 
+		// set gitMergeInProgress context
+		commands.executeCommand('setContext', 'gitMergeInProgress', merge.length > 0);
```

Comment:
This isn't 100% correct. You can be in the middle of a merge operation and already have resolved all conflicts. You need to figure out how to detect that a repository is in the _middle of a merge_ state, just like we do for rebase with `getRebaseCommit`.

Reply:
Switched to using the existence of the `MERGE_HEAD` file to detect that a merge is in progress.